<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkAstrosticksModelParametersWidgetControls</class>
 <widget class="QWidget" name="QmitkAstrosticksModelParametersWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>106</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_RandomCheck">
     <property name="toolTip">
      <string>Use random number and orientation of sticks.</string>
     </property>
     <property name="text">
      <string>Randomize Sticks</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;T2&lt;/span&gt; - Relaxation:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_T2box">
        <property name="toolTip">
         <string>T2 relaxation time of this compartment (in milliseconds).</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;d&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="m_D1box">
        <property name="toolTip">
         <string>Diffusivity along sticks.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
        <property name="value">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_T2box</tabstop>
  <tabstop>m_D1box</tabstop>
  <tabstop>m_RandomCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
